models/SLICOT/beam/beam.jl

```diff
@@ -3,13 +3,15 @@ Model: Beam (348 variables, 1 input)
 =#
 using Reachability, MAT, SparseArrays
 
-beam(o::Pair{Symbol, <:Any}...) = beam(Options(Dict{Symbol, Any}(o)))
 
-function beam(input_options::Options)
+#get_filename() = @relpath "beam.mat"
+
+function beam(input_options=nothing; filename=@relpath "beam.mat")
```

Comment:
```suggestion
function beam(input_options=nothing::Options=Options(); filename=@relpath "beam.mat")
```
and remove the `if` below (merge in any case).